src/tap/README.md

```diff
@@ -24,6 +24,31 @@ tapcell [-tapcell_master tapcell_master]
         [-endcap_prefix endcap_prefix]
 ```
 
+- `-tapcell_master`. Specify the master used as a tapcell.
+- `-endcap_master`. Specify the master used as an endcap.
+- `-distance`. Specify the distance between each tapcell in the checkerboard.
+- `-halo_width_x`. Specify the horizontal halo size around macros during cut rows.
+- `-halo_width_y`. Specify the vertical halo size around macros during cut rows.
+- `-tap_nwin2_master`, `-tap_nwin3_master`, `-tap_nwout2_master`,
+`-tap_nwout3_master`, `-tap_nwintie_master`, `-tap_nwouttie_master`.
+Specify the master cells placed at the top and bottom of macros and of the core area,
+according the row orientation.
+- `-incnrcap_nwin_master`, `-incnrcap_nwout_master`. Specify the master cells
+placed at the corners of macros.
+- `-cnrcap_nwin_master`, `-cnrcap_nwout_master`. Specify the macro cells placed
+at the corners of the core area, according the row orientation.
```

Comment:
Maybe don't compound the similar options? i.e., one option per line -- it might be repetitive but clear?
Or at least say somewhere what the abbrev. mean, e.g., nwin/nwout, what does the number mean? why start at 2?